docs/agent-oriented-approach.md

```diff
@@ -0,0 +1,49 @@
+
+<!--As their name suggest, there are more than one agent in a multi-agent environment, each changing the state of the environment to their own liking. Crucially, as a decentralised system, MAS makes no such assumption that agents are owned by the same entity, or designed homogeneously along the same line of standards and principles. In fact, a distinctive characteristic of a multi-agent environment is that it is inhabited by more than one agent, each being owned by a different stakeholder (individual, company, government).-->
+
+As such, a distinctive characteristic of a multi-agent environment is that it is inhabited by more than one agent, as the name suggests :), where each agent may be owned potentially by a different stakeholder (individual, company, government). Since by design each agent represents and looks after the interests of its owner(s), and because different stakeholders may have unaligned, conflicting, or contradictory interests, it is very common that in a multi-agent system, the agents' objectives, values and preferences may also be unaligned, conflicting, or contradictory.
+
+There are practical implications that follow from the above when it comes to designing an agent. For example, it is not rational for an agent to automatically rely on the information it receives from other agents. The information could be incomplete; what is unrevealed may have been deemed private for strategic reasons. The information could be uncertain, the result of an inaccurate prediction. Or it could be an outright lie, due to the adversarial nature of the environment. Therefore there is a degree of uncertainty attached to almost all information an agent receives in a multi-agent system. One could argue for taking a sceptical approach; treating everything as uncertain, unless proved otherwise.  
+
+Therefore when designing an agent, one has to be aware of this conflicts of interest, and take this into account when for example the agent gathers information from other agents .    
+
+## Asynchronization
+
+Another inherent difference: Agents are autonomous, thus:
+
+**Asynchronization**; agent vs object (want vs must); no guarantees w.r.t. Responses (may lie/incomplete (see above), respond later (you are low priority to them), don’t respond because they don’t want to)
+
+In practice: your decision making and action should be as much as possible independent of interactions with others; decoupling of decision/actions and interactions. Try to interact with others but decide/act based on information/responses you have and shouldn’t wait to get all responses
+
+## Complex, Incomplete, Inconsistent and Uncertain
```

Comment:
This section somewhat repeats what is at the end of `Conflicting Environment`. I'd say it can be dropped, but incorporating the following into that other section:

* `Complex` (because this isn't necessarily mentioned)
* `So should build an approximate model based on all the information one has.`